import_cambridge_lands.py

```diff
@@ -26,7 +26,9 @@ def __init__(
             self.lr_api_url = self.lr_api_url + '/'
 
     def get_lr_data(self, uprn):
-        url = '{0}{1}'.format(self.lr_api_url, uprn)
+        # some gazetteer database applications will pad UPRNs that are less
+        # than 12 digits with zeros BUT the HMRC API needs you to remove them
+        url = '{0}{1}'.format(self.lr_api_url, int(uprn))
```

Comment:
Yes, I'm not discussing the fact that HMRC API requires the zeros to be truncated, I'm saying that there is probably a reason why we need to keep them